lemmy_api/src/community.rs

```diff
@@ -129,15 +130,31 @@ impl Perform for CreateCommunity {
       return Err(APIError::err("community_already_exists").into());
     }
 
+    // Check to make sure the icon and banners are urls
+    let icon = Some(data.icon.to_owned());
+    let banner = Some(data.banner.to_owned());
+
+    if let Some(Some(icon)) = &icon {
```

Comment:
Putting this before the `let icon` means you only need one `Some` here ;)

Reply:
Unfortunately tho that would mean I would have to use `icon: Some(icon)` below instead of just `icon,`